### DISCUSSION TITLE
Minio `make build` with CGO_DISABLED=0

### DISCUSSION THREAD

**Discussion by lpoli** [minio/minio#13982] [Q&A] [answered]
Why do `make build` has `CGO_ENABLED=0`?
I have written a new gateway for [0chain](https://0chain.net/) and wanted to send PR. One of the module depends on  https://github.com/herumi/bls-go-binary which compiles with `CGO_ENABLED=1`.

Is there any workaround to pass `make test` and send PR?

**Comment by klauspost** [comment, marked as answer]
We do not allow cgo in MinIO. Using cgo has several negative side-effects which we do not want. 

You will need to implement it without using cgo.
